### PR TITLE
log.h: adjust alignment/layout of log messages

### PR DESCRIPTION
**Pull request by justinmk**
Example log messages:
- New: `2016/09/27 18:00:49 INFO  28117/tui_flush:597: ...`
- Old:   `2016/09/27 18:00:49 [info @ tui_flush:597] 28117 - ...`

---

Other changes in this PR:
- move `ui_bridge.{c,h}` into `tui/`
- random comment tweaks

### CODE REVIEW

File: src/nvim/diff.c [hunk 1]
```diff
@@ -906,17 +906,15 @@ void ex_diffpatch(exarg_T *eap)
     eval_patch((char *) tmp_orig, (char *) eap->arg, (char *) tmp_new);
 #endif  // ifdef UNIX
   } else {
-    // Build the patch command and execute it. Ignore errors. Switch to
-    // cooked mode to allow the user to respond to prompts.
+    // Build the patch command and execute it. Ignore errors.
```
Comment (justinmk) [inline src/nvim/diff.c]: cooked mode is a non-sequitur for nvim

File: src/nvim/tui/tui.c [hunk 2]
```diff
@@ -17,13 +19,13 @@
...
-#include "nvim/tui/tui.h"
-#include "nvim/tui/input.h"
 #include "nvim/os/input.h"
 #include "nvim/os/os.h"
 #include "nvim/strings.h"
 #include "nvim/ugrid.h"
-#include "nvim/ui_bridge.h"
+#include "nvim/tui/input.h"
+#include "nvim/tui/tui.h"
+#include "nvim/tui/ui_bridge.h"
```
Comment (justinmk) [inline src/nvim/tui/tui.c]: Moved `ui_bridge.h` from top-level, into the `tui/` namespace.

**Review by jamessan — APPROVED** [review]

File: src/nvim/event/queue.c [hunk 3]
```diff
@@ -26,20 +26,20 @@
 //    +-----------+   +-----------+    +---------+    +---------+
 //
 //
-// In the above diagram, the lower boxes represents event emitters, each with
-// it's own private queue that have the event loop queue as the parent.
+// The lower boxes represents event emitters, each with its own private queue
```
Comment (brcolow) [inline src/nvim/event/queue.c]: represents => represent

File: src/nvim/event/queue.c [hunk 4]
```diff
@@ -26,20 +26,20 @@
...
+// having the event loop queue as the parent.
 //
 // When idle, the main loop spins the event loop which queues events from many
```
Comment (rajeshceg3) [inline src/nvim/event/queue.c]: Too many the's in one line ? the main loop => main loop

**Pull request merged**